torch/utils/benchmark/utils/timer.py

```diff
@@ -21,6 +27,64 @@ def timer() -> float:
     timer = timeit.default_timer
 
 
+PYTHON = "Python"
+CPP = "C++"
+
+
+class InnerTimer(Protocol):
```

Comment:
Would be nice to have an explanation what this is, and what types are known to implement this Protocol.